config/webpack/webpack.config.js

```diff
@@ -178,20 +178,24 @@ const buildWebpackConfigs = builds.map(
             },
             {
               test: /\.less$/,
-              exclude: /node_modules/,
-              use: ExtractTextPlugin.extract({
-                fallback: 'style-loader',
-                use: makeCssLoaders()
-              })
+              oneOf: [
+                ...paths.compilePackages.map(packageName => ({
+                  test: /\.less$/,
```

Comment:
This test is redundant now right?

Reply:
Correct. All 4 inner tests were redundant and have now been removed and re-tested. Thanks